.github/pull_request_template.md

```diff
@@ -2,10 +2,12 @@
 
 ### Description :
 
-describe changes made in code
+<!--Describe changes made in code.-->
```

Comment:
Thanks @anishagg17 for the pr, but I think you should squash your commits as the commits belong to the same issue so its better to have a concise commit for every issue